manager.go

```diff
@@ -32,7 +33,9 @@ type SpinnerManager interface {
 }
 
 type spinnerManager struct {
-	spinners      []*Spinner
+	spinners   []*Spinner
+	spinnersMu sync.RWMutex
```

Comment:
Small nit: Could this be renamed to `mutex` so it is consistent with the setup in `spinner.go`?

Reply:
Sure thing - it's pretty standard if there are multiple mutexes to name them like that (after the field they protect) since you want to be as granular as possible when using multiple on a single struct. Totally happy to rename it here though